### DISCUSSION TITLE
react-native-skia new architecture adoption

### DISCUSSION THREAD

**Discussion by leotm** [reactwg/react-native-new-architecture#55] [Libraries]
- https://github.com/Shopify/react-native-skia/issues/387
- https://github.com/Shopify/react-native-skia/issues/676

Not yet started but raised ^ may try migrating soon once fingers in less pies - anyone else welcome 😃

**Comment by cipolleschi** [comment]
Hi @leotm! Do you have any update on this migration?

**Reply by leotm** [reply]
didn't get round to this in the end, but seeing the @Shopify team got a handle on it
@chrfalch looks like `🎬 Started` or `🏃‍♂️ Ongoing` would be appropriate?
couldn't find much more checking Issues/PRs/Releases/Discussions
